<!doctype html> <html lang=en > <script async src="https://www.googletagmanager.com/gtag/js?id=G-MGPKNNYFVQ"></script> <script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-MGPKNNYFVQ'); </script> <meta charset=UTF-8 > <meta name=viewport  content="width=device-width, initial-scale=1"> <link rel=stylesheet  href="/css/franklin.css"> <link rel=stylesheet  href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700,400italic|Source+Code+Pro:400,700" type="text/css"> <link rel=stylesheet  href="/css/font-awesome.min.css"> <link rel=stylesheet  href="/css/celeste.min.css"> <link rel=icon  type="image/png" sizes=192x192  href="/assets/icon.png"> <link rel="shortcut icon" href="/assets/icon.png"> <link rel=apple-touch-icon-precomposed  sizes=152x152  href="/assets/icon.png"> <title>Joe Greener</title> <header> <nav class=nav-main > <ul> <li class=hvr-underline-reveal ><a href="/">Home</a> <li class=hvr-underline-reveal ><a href="/posts/">Posts</a> <li class=hvr-underline-reveal ><a href="/contact/">Contact</a> </ul> </nav> </header> <div class=franklin-content ><h1 id=joe_greener ><a href="#joe_greener" class=header-anchor >Joe Greener</a></h1> <p>I am a group leader at the <a href="https://www2.mrc-lmb.cam.ac.uk">MRC Laboratory of Molecular Biology</a> working on improving molecular simulations. My group page is <a href="https://www2.mrc-lmb.cam.ac.uk/groups/greener">here</a> and my entry on the LMB website is <a href="https://www2.mrc-lmb.cam.ac.uk/group-leaders/a-to-g/joe-greener">here</a>. I am currently looking to grow the group. You can also find me on <a href="https://twitter.com/jgreener64">X</a>, <a href="https://bsky.app/profile/jgreener64.bsky.social">Bluesky</a> and <a href="https://github.com/jgreener64">GitHub</a>.</p> <p>My interests include molecular simulation, protein structures, software development and open science. Currently my work aims to improve the molecular mechanics force fields used to simulate biomolecules with machine learning techniques. I am also interested in developing new types of machine learning force fields that are fast enough to simulate biomolecules.</p> <p>Previously I was a research fellow &#40;postdoc&#41; in the <a href="http://bioinf.cs.ucl.ac.uk/introduction">Bioinformatics Group</a> at <a href="https://www.ucl.ac.uk">University College London</a> &#40;UCL&#41; and a PhD student in the <a href="http://www.sbg.bio.ic.ac.uk/index.html">Structural Bioinformatics Group</a> at <a href="https://www.imperial.ac.uk">Imperial College London</a>.</p> <div class=row > <div class=container > <img src="/assets/protein.png" alt="Modulation of adenylate cyclase"> </div> </div> <h1 id=publications ><a href="#publications" class=header-anchor >Publications</a></h1> <p><a href="https://scholar.google.co.uk/citations?view_op&#61;list_works&amp;hl&#61;en&amp;user&#61;IG6H3tIAAAAJ">Google Scholar page</a></p> <p><strong>Greener JG</strong>. Reversible molecular simulation for training classical and machine learning force fields, <em>Proceedings of the National Academy of Sciences</em> 122&#40;22&#41;, e2426058122 &#40;2025&#41; - <a href="https://www.pnas.org/doi/10.1073/pnas.2426058122">link</a> - <a href="https://github.com/greener-group/rev-sim">code</a></p> <p><strong>Greener JG</strong> and Jamali K. Fast protein structure searching using structure graph embeddings, <em>Bioinformatics Advances</em> 5:1, vbaf042 &#40;2025&#41; - <a href="https://academic.oup.com/bioinformaticsadvances/article/5/1/vbaf042/8107707">link</a> - <a href="https://github.com/greener-group/progres">code</a> - <a href="https://progres.mrc-lmb.cam.ac.uk">server</a></p> <p>Yu CWH, Fischer ES, <strong>Greener JG</strong>, Yang J, Zhang Z, Freund SMV and Barford D. Molecular mechanism of Mad2 conformational conversion promoted by the Mad2-interaction motif of Cdc20, <em>Protein Science</em> 34&#40;4&#41;, e70099 &#40;2025&#41; - <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pro.70099">link</a></p> <p>Wang Y, Takaba K, Chen MS, Wieder M, Xu Y, Zhu T, Zhang JZH, Nagle A, Yu K, Wang X, Cole DJ, Rackers JA, Cho K, <strong>Greener JG</strong>, Eastman P, Martiniani S and Tuckerman ME. On the design space between molecular mechanics and machine learning force fields, <em>Applied Physics Reviews</em> 12, 021304 &#40;2025&#41; - <a href="https://pubs.aip.org/aip/apr/article/12/2/021304/3342041/On-the-design-space-between-molecular-mechanics">link</a></p> <p><strong>Greener JG</strong>. Differentiable simulation to develop molecular dynamics force fields for disordered proteins, <em>Chemical Science</em> 15, 4897-4909 &#40;2024&#41; - <a href="https://pubs.rsc.org/en/content/articlelanding/2024/sc/d3sc05230c">link</a> - <a href="https://github.com/greener-group/GB99dms">code</a></p> <p>Roesch E, <strong>Greener JG</strong>, MacLean AL, Nassar H, Rackauckas C, Holy TE and Stumpf MPH. Julia for biologists, <em>Nature Methods</em> 20, 655-664 &#40;2023&#41; - <a href="https://www.nature.com/articles/s41592-023-01832-z">link</a></p> <p><strong>Greener JG&#42;</strong>, Kandathil SM&#42;, Moffat L and Jones DT. A guide to machine learning for biologists, <em>Nature Reviews Molecular Cell Biology</em> 23, 40-55 &#40;2022&#41; - <a href="https://www.nature.com/articles/s41580-021-00407-0">link</a> - <a href="https://rdcu.be/cx9lv">open link</a></p> <p>Kandathil SM, <strong>Greener JG</strong>, Lau AM and Jones DT. Ultrafast end-to-end protein structure prediction enables high-throughput exploration of uncharacterized proteins, <em>Proceedings of the National Academy of Sciences</em> 119&#40;4&#41;, e2113348119 &#40;2022&#41; - <a href="https://www.pnas.org/doi/full/10.1073/pnas.2113348119">link</a> - <a href="https://github.com/psipred/DMPfold2">code</a></p> <p>Moffat L, <strong>Greener JG</strong> and Jones DT. Using AlphaFold for Rapid and Accurate Fixed Backbone Protein Design, <em>bioRxiv</em> &#40;2021&#41; - <a href="https://www.biorxiv.org/content/10.1101/2021.08.24.457549">link</a></p> <p><strong>Greener JG</strong> and Jones DT. Differentiable molecular simulation can learn all the parameters in a coarse-grained force field for proteins, <em>PLoS ONE</em> 16&#40;9&#41;, e0256990 &#40;2021&#41; - <a href="https://journals.plos.org/plosone/article?id&#61;10.1371/journal.pone.0256990">link</a> - <a href="https://github.com/psipred/cgdms">code</a></p> <p><strong>Greener JG&#42;</strong>, Desai N&#42;, Kandathil SM and Jones DT. Near-complete protein structural modelling of the minimal genome, <em>arXiv</em> 2007.06623 &#40;2020&#41; - <a href="https://arxiv.org/abs/2007.06623">link</a></p> <p><strong>Greener JG</strong>, Selvaraj J and Ward BJ. BioStructures.jl: read, write and manipulate macromolecular structures in Julia, <em>Bioinformatics</em> 36&#40;14&#41;, 4206-4207 &#40;2020&#41; - <a href="https://academic.oup.com/bioinformatics/article/36/14/4206/5837108">link</a> - <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa502/5837108?guestAccessKey&#61;aec90643-1d43-4521-9883-4a4a669187da">open link</a> - <a href="https://github.com/BioJulia/BioStructures.jl">code</a></p> <p><strong>Greener JG&#42;</strong>, Kandathil SM&#42; and Jones DT. Deep learning extends de novo protein modelling coverage of genomes using iteratively predicted structural constraints, <em>Nature Communications</em> 10:3977 &#40;2019&#41; - <a href="https://www.nature.com/articles/s41467-019-11994-0">link</a> - <a href="http://bioinf.cs.ucl.ac.uk/psipred">website</a> - <a href="https://github.com/psipred/DMPfold">code</a></p> <p>Kandathil SM, <strong>Greener JG</strong> and Jones DT. Recent Developments in Deep Learning Applied to Protein Structure Prediction, <em>Proteins</em> 87&#40;12&#41;, 1179-1189 &#40;2019&#41; - <a href="https://onlinelibrary.wiley.com/doi/10.1002/prot.25824">link</a></p> <p>Kandathil SM, <strong>Greener JG</strong> and Jones DT. Prediction of inter-residue contacts with DeepMetaPSICOV in CASP13, <em>Proteins</em> 87&#40;12&#41;, 1092-1099 &#40;2019&#41; - <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/prot.25779">link</a> - <a href="https://github.com/psipred/DeepMetaPSICOV">code</a></p> <p><strong>Greener JG&#42;</strong>, Moffat L&#42; and Jones DT. Design of metalloproteins and novel protein folds using variational autoencoders, <em>Scientific Reports</em> 8:16189 &#40;2018&#41; - <a href="https://www.nature.com/articles/s41598-018-34533-1">link</a> - <a href="https://github.com/psipred/protein-vae">code</a></p> <p>Craven GB, Affron DP, Allen CE, Matthies S, <strong>Greener JG</strong>, Morgan RML, Tate EW, Armstrong A and Mann DJ. High-Throughput Kinetic Analysis for Target-Directed Covalent Ligand Discovery, <em>Angewandte Chemie International Edition</em> 57&#40;19&#41;, 5257-5261 &#40;2018&#41; - <a href="http://onlinelibrary.wiley.com/doi/10.1002/anie.201711825/full">link</a></p> <p><strong>Greener JG</strong> and Sternberg MJE. Structure-based prediction of protein allostery, <em>Current Opinion in Structural Biology</em> 50, 1-8 &#40;2018&#41; - <a href="http://www.sciencedirect.com/science/article/pii/S0959440X17301252">link</a></p> <p><strong>Greener JG</strong>, Filippis I and Sternberg MJE. Predicting protein dynamics and allostery using multi-protein atomic distance constraints, <em>Structure</em> 25, 546-558 &#40;2017&#41; - <a href="http://www.cell.com/structure/fulltext/S0969-2126&#40;17&#41;30008-4">link</a> - <a href="https://github.com/jgreener64/ProteinEnsembles.jl">code</a></p> <p><strong>Greener JG</strong> and Sternberg MJE. AlloPred: prediction of allosteric pockets on proteins using normal mode perturbation analysis, <em>BMC Bioinformatics</em> 16:335 &#40;2015&#41; - <a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0771-1">link</a> - <a href="http://www.sbg.bio.ic.ac.uk/allopred/home">website</a> - <a href="https://github.com/jgreener64/allopred">code</a></p> <h1 id=software ><a href="#software" class=header-anchor >Software</a></h1> <p>Some software I have written or contributed to:</p> <ul> <li><p><a href="https://github.com/JuliaMolSim/Molly.jl">Molly.jl</a> - molecular simulation in Julia</p> <li><p><a href="https://github.com/greener-group/progres">Progres</a> - fast protein structure searching</p> <li><p><a href="https://github.com/BioJulia/BioStructures.jl">BioStructures.jl</a> - a Julia package to read, write and manipulate macromolecular structures</p> <li><p><a href="https://github.com/jgreener64/mmterm">mmterm</a> - view proteins and trajectories in the terminal</p> <li><p><a href="https://github.com/jgreener64/Bio3DView.jl">Bio3DView.jl</a> - view macromolecular structures from Julia</p> <li><p><a href="https://github.com/jgreener64/pdb-benchmarks">pdb-benchmarks</a> - benchmarking common tasks on proteins in various languages and packages</p> <li><p><a href="https://github.com/psipred/cgdms">cgdms</a> - differentiable molecular simulation of proteins with a coarse-grained potential</p> <li><p><a href="https://github.com/psipred/DMPfold">DMPfold</a> and <a href="https://github.com/psipred/DMPfold2">DMPfold2</a> - de novo protein structure prediction</p> <li><p><a href="https://github.com/JuliaMolSim/AtomsBase.jl">AtomsBase.jl</a> - a Julian abstract interface for atomic structures</p> <li><p><a href="http://biopython.org">Biopython</a> - biology tools in Python</p> </ul> <p>There is more software associated with a few of the papers listed above.</p> <h1 id=talks ><a href="#talks" class=header-anchor >Talks</a></h1> <p>CECAM Summer School on Sampling High-Dimensional Probability Measures with Applications in &#40;Non&#41;Equilibrium Molecular Dynamics and Statistics &#40;Jul 2025&#41; - <a href="https://docs.google.com/presentation/d/1CvGwFn0XjiIfAQ562RlIPGjXRKnsNXLYjmPRDFS7SQs/edit?usp&#61;sharing">slides</a></p> <p>SBS Bioinformatics Seminar &#40;Apr 2025&#41; - <a href="https://docs.google.com/presentation/d/1_YNIXh_gzwx0_yOx1GZXbroHsPwA_uq2cYEXRfGLiNM/edit?usp&#61;sharing">slides</a></p> <p>Cambridge Cheminformatics Meet &#40;Feb 2025&#41; - <a href="https://youtu.be/Hd_KuxXLCUg?si&#61;hoSzTnSO9Q-ubV_7&amp;t&#61;2078">video</a> - <a href="https://docs.google.com/presentation/d/1pxtdKks8JlgXbs6qb_LQDGjq9l_N5iqWAfydXeBh-hE/edit?usp&#61;sharing">slides</a></p> <p>MolSSI workshop on Julia for Computational Molecular and Materials Science &#40;Oct 2024&#41; - <a href="https://docs.google.com/presentation/d/1KSs22jJL2lYCQE3qKgEuZGYLLCoLB7XpRpN4jJu4a6I/edit?usp&#61;sharing">slides</a></p> <p>Computational Molecular Science 2024 &#40;Sep 2024&#41; - <a href="https://docs.google.com/presentation/d/1IJCnm6Q3XAN-CxlcEYXPuzPxLvfGp_EKnozutMI29Dg/edit?usp&#61;sharing">slides</a></p> <p>JuliaCon 2024 &#40;Jul 2024&#41; - <a href="https://www.youtube.com/watch?v&#61;XTE5HSo-jx8">video</a> - <a href="https://docs.google.com/presentation/d/1E7Ez4WQAOfizXirQdsILZp1UZfgV0U_BBoxEQm86ZVE/edit?usp&#61;sharing">slides</a></p> <p>Enzyme Conference 2024 &#40;Mar 2024&#41; - <a href="https://www.youtube.com/live/fRZwgVbcdv8?si&#61;xtBI_mNkLewFSgdG&amp;t&#61;2641">video</a> - <a href="https://docs.google.com/presentation/d/1az51CuXO-X3vDraJrASUK5Vn5fK8r2ijLJPvBze0sfE/edit?usp&#61;sharing">slides</a></p> <p>Talks and practical at SINEQ summer school, Ã‰cole des ponts ParisTech &#40;Sep 2023&#41; - <a href="https://docs.google.com/presentation/d/1SjzRi7jFbgFwP9kupwdtkxmr2x0OWjzzXwnoenCcCQg/edit?usp&#61;sharing">slides</a> - <a href="https://github.com/noeblassel/SINEQSummerSchool2023/blob/main/notebooks/molly_average.ipynb">notebook</a></p> <p>Talk at Institute of Genetics and Cancer, University of Edinburgh &#40;Mar 2023&#41; - <a href="https://docs.google.com/presentation/d/11naV12vWIZ3TlmaX8-IB4nx6vUOwLXdcvZWABLPd09A/edit?usp&#61;sharing">slides</a></p> <p>Enzyme Conference 2023 &#40;Feb 2023&#41; - <a href="https://www.youtube.com/watch?v&#61;6G97jDVPlYc">video</a> - <a href="https://docs.google.com/presentation/d/1xsi2tbOJ3ceHu8-4izIMmo6M5EfnU1BrVwEtRBybZIM/edit?usp&#61;sharing">slides</a></p> <p>Crick Christmas Lecture 2022 &#40;Dec 2022&#41; - <a href="https://docs.google.com/presentation/d/1ilWqWmYNuuMeO-vhJqukBaGfW9FXa7WeFyykKuZmmzc/edit?usp&#61;sharing">slides</a></p> <p>JuliaMolSim minisymposium at JuliaCon 2022 &#40;Jul 2022&#41; - <a href="https://youtu.be/trapn-yIv8g?t&#61;1889">video</a> - <a href="https://docs.google.com/presentation/d/1Wkurom5G2N_qwqNueGwLCyYmpETT6dyeuW6UfUlrb5A/edit?usp&#61;sharing">slides</a></p> <p>8th Annual CCP-BioSim Conference invited talk &#40;Jun 2022&#41; - <a href="https://docs.google.com/presentation/d/1thJOrj8kAWKxLRWAEcDZ1tng0d2E4IWIyfJ4yot6wPs/edit?usp&#61;sharing">slides</a></p> <p>Lennard Jones Centre &#40;May 2022&#41; - <a href="https://www.youtube.com/watch?v&#61;0MLl6mQWtEQ">video</a> - <a href="https://docs.google.com/presentation/d/1jf9Zt58D1TWdc9p6YPxtGRNtlddvkSr8shWU5rTgIFM/edit?usp&#61;sharing">slides</a></p> <p>Scientific Machine Learning Webinar Series &#40;Mar 2022&#41; - <a href="https://www.cmu.edu/aced/sciML.html">video</a> - <a href="https://docs.google.com/presentation/d/1Ukvmyt2tUPmGV6ryIgvuGMnJy5-b9awlmiPJfdMm4uQ/edit?usp&#61;sharing">slides</a></p> <p>Microsoft Research New England ML Ideas Lunch &#40;Mar 2021&#41; - <a href="https://docs.google.com/presentation/d/1FkSsUWMBQTbiGhc1u30qRcj25CslL5xnt9gAyqIcRjM/edit?usp&#61;sharing">slides</a></p> <p>Imperial College Computational Biology Society &#40;Nov 2020&#41; - <a href="https://docs.google.com/presentation/d/1zkJ8UwSwu2nsjIwHvIiTXL2z4fyxs_kJDjxjwoB8vBo/edit?usp&#61;sharing">slides</a></p> <p>KQ Codes Technical Social &#40;Jan 2020&#41; - <a href="https://docs.google.com/presentation/d/1wWdqTR1Ifls-psjcj-Euv6LY7Z5LWBei7VsiqPlB46E/edit?usp&#61;sharing">slides</a>.</p> <p>3DSIG meeting at ISMB &#40;Jul 2019&#41; - <a href="https://www.youtube.com/watch?v&#61;apv5YPM0HqY&amp;list&#61;PLmX8XnLr6zeHO2l7Gki56qjGxhhHYljin&amp;index&#61;17">video</a> - <a href="https://docs.google.com/presentation/d/1akuaq1QHbCVdxshmrtbJm6xssRZ3PwOYcR0PviU6YWg/edit?usp&#61;sharing">slides</a>.</p> <p>London Julia <a href="https://www.meetup.com/London-Julia-User-Group/events/262441283">Meetup group</a> &#40;Jun 2019&#41; - <a href="https://docs.google.com/presentation/d/13KxvnwR-XFMFyUSth1hsXSoW0Zx5XLzds6fci7RUUDQ/edit?usp&#61;sharing">slides</a>.</p> <p>Friday Wrap talk at Institute of Structural and Molecular Biology &#40;May 2019&#41; - <a href="https://docs.google.com/presentation/d/1PAagp9hPXTT3ks1iOvFKOyw0TOD_ZyF9dSmvoVVudVg/edit?usp&#61;sharing">slides</a>.</p> <p>3DSIG meeting at ISMB &#40;Jul 2016&#41;.</p> <p>Bioinformatics London <a href="https://www.meetup.com/Bioinformatics-London/events/230436340">Meetup group</a> &#40;Apr 2016&#41;.</p> <h1 id=awards ><a href="#awards" class=header-anchor >Awards</a></h1> <p>Warren DeLano Structural Bioinformatics and Computational Biophysics Award for best presentation at 3DSIG 2016.</p> <p>Awarded ISMB/ECCB Travel Fellowship to attend ISMB 2019 by 3DSIG and ISCB.</p> <p>Prize for highest mark in MSc class &#40;2014&#41;.</p> <p>Runner-up prize for best presentation at Imperial College London Department of Life Sciences Research Day &#40;2017&#41;.</p> <div class=page-foot > <a href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a> Joe Greener. Last modified: October 15, 2025. Website built with <a href="https://github.com/tlienart/Franklin.jl">Franklin.jl</a> and the <a href="https://julialang.org">Julia programming language</a>. </div> </div>