@def title = "Joe Greener"

# Joe Greener

I am a group leader at the [MRC Laboratory of Molecular Biology](https://www2.mrc-lmb.cam.ac.uk) working on improving molecular simulations. My group page is [here](https://www2.mrc-lmb.cam.ac.uk/groups/greener) and my entry on the LMB website is [here](https://www2.mrc-lmb.cam.ac.uk/group-leaders/a-to-g/joe-greener). I am currently looking to grow the group. You can also find me on [Twitter](https://twitter.com/jgreener64) and [GitHub](https://github.com/jgreener64).

My interests include molecular simulation, protein structures, software development and open science. Currently my work aims to use the concept of differentiable programming to improve the force fields used to simulate biomolecules, enabling us to better study disordered proteins and protein-ligand interactions.

Previously I was a research fellow (postdoc) in the [Bioinformatics Group](http://bioinf.cs.ucl.ac.uk/introduction) at [University College London](https://www.ucl.ac.uk) (UCL) and a PhD student in the [Structural Bioinformatics Group](http://www.sbg.bio.ic.ac.uk/index.html) at [Imperial College London](https://www.imperial.ac.uk).

~~~
<div class="row">
  <div class="container">
    <img src="/assets/protein.png" alt="Modulation of adenylate cyclase">
  </div>
</div>
~~~

# Publications

[Google Scholar page](https://scholar.google.co.uk/citations?view_op=list_works&hl=en&user=IG6H3tIAAAAJ)

Yu CWH, Fischer ES, __Greener JG__, Yang J, Zhang Z, Freund SMV, Barford D. Molecular mechanism of Mad2 conformational conversion promoted by the Mad2-interaction motif of Cdc20, *bioRxiv* (2024) - [link](https://www.biorxiv.org/content/10.1101/2024.03.03.583158)

__Greener JG__. Differentiable simulation to develop molecular dynamics force fields for disordered proteins, *Chemical Science* (2024) - [link](https://pubs.rsc.org/en/content/articlelanding/2024/sc/d3sc05230c) - [code](https://github.com/greener-group/GB99dms)

Roesch E, __Greener JG__, MacLean AL, Nassar H, Rackauckas C, Holy TE and Stumpf MPH. Julia for biologists, *Nature Methods* 20, 655-664 (2023) - [link](https://www.nature.com/articles/s41592-023-01832-z)

__Greener JG__ and Jamali K. Fast protein structure searching using structure graph embeddings, *bioRxiv* (2022) - [link](https://www.biorxiv.org/content/10.1101/2022.11.28.518224v1) - [code](https://github.com/jgreener64/progres)

__Greener JG\*__, Kandathil SM\*, Moffat L and Jones DT. A guide to machine learning for biologists, *Nature Reviews Molecular Cell Biology* 23, 40-55 (2022) - [link](https://www.nature.com/articles/s41580-021-00407-0) - [open link](https://rdcu.be/cx9lv)

Kandathil SM, __Greener JG__, Lau AM and Jones DT. Ultrafast end-to-end protein structure prediction enables high-throughput exploration of uncharacterized proteins, *Proceedings of the National Academy of Sciences* 119(4), e2113348119 (2022) - [link](https://www.pnas.org/doi/full/10.1073/pnas.2113348119) - [code](https://github.com/psipred/DMPfold2)

Moffat L, __Greener JG__ and Jones DT. Using AlphaFold for Rapid and Accurate Fixed Backbone Protein Design, *bioRxiv* (2021) - [link](https://www.biorxiv.org/content/10.1101/2021.08.24.457549v1)

__Greener JG__ and Jones DT. Differentiable molecular simulation can learn all the parameters in a coarse-grained force field for proteins, *PLoS ONE* 16(9), e0256990 (2021) - [link](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0256990) - [code](https://github.com/psipred/cgdms)

__Greener JG\*__, Desai N\*, Kandathil SM and Jones DT. Near-complete protein structural modelling of the minimal genome, *arXiv* 2007.06623 (2020) - [link](https://arxiv.org/abs/2007.06623)

__Greener JG__, Selvaraj J and Ward BJ. BioStructures.jl: read, write and manipulate macromolecular structures in Julia, *Bioinformatics* 36(14), 4206-4207 (2020) - [link](https://academic.oup.com/bioinformatics/article/36/14/4206/5837108) - [open link](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa502/5837108?guestAccessKey=aec90643-1d43-4521-9883-4a4a669187da) - [code](https://github.com/BioJulia/BioStructures.jl)

__Greener JG\*__, Kandathil SM\* and Jones DT. Deep learning extends de novo protein modelling coverage of genomes using iteratively predicted structural constraints, *Nature Communications* 10:3977 (2019) - [link](https://www.nature.com/articles/s41467-019-11994-0) - [website](http://bioinf.cs.ucl.ac.uk/psipred) - [code](https://github.com/psipred/DMPfold)

Kandathil SM, __Greener JG__ and Jones DT. Recent Developments in Deep Learning Applied to Protein Structure Prediction, *Proteins* 87(12), 1179-1189 (2019) - [link](https://onlinelibrary.wiley.com/doi/10.1002/prot.25824)

Kandathil SM, __Greener JG__ and Jones DT. Prediction of inter-residue contacts with DeepMetaPSICOV in CASP13, *Proteins* 87(12), 1092-1099 (2019) - [link](https://onlinelibrary.wiley.com/doi/full/10.1002/prot.25779) - [code](https://github.com/psipred/DeepMetaPSICOV)

__Greener JG\*__, Moffat L\* and Jones DT. Design of metalloproteins and novel protein folds using variational autoencoders, *Scientific Reports* 8:16189 (2018) - [link](https://www.nature.com/articles/s41598-018-34533-1) - [code](https://github.com/psipred/protein-vae)

Craven GB, Affron DP, Allen CE, Matthies S, __Greener JG__, Morgan RML, Tate EW, Armstrong A and Mann DJ. High-Throughput Kinetic Analysis for Target-Directed Covalent Ligand Discovery, *Angewandte Chemie International Edition* 57(19), 5257-5261 (2018) - [link](http://onlinelibrary.wiley.com/doi/10.1002/anie.201711825/full)

__Greener JG__ and Sternberg MJE. Structure-based prediction of protein allostery, *Current Opinion in Structural Biology* 50, 1-8 (2018) - [link](http://www.sciencedirect.com/science/article/pii/S0959440X17301252)

__Greener JG__, Filippis I and Sternberg MJE. Predicting protein dynamics and allostery using multi-protein atomic distance constraints, *Structure* 25, 546-558 (2017) - [link](http://www.cell.com/structure/fulltext/S0969-2126(17)30008-4) - [code](https://github.com/jgreener64/ProteinEnsembles.jl)

__Greener JG__ and Sternberg MJE. AlloPred: prediction of allosteric pockets on proteins using normal mode perturbation analysis, *BMC Bioinformatics* 16:335 (2015) - [link](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0771-1) - [website](http://www.sbg.bio.ic.ac.uk/allopred/home) - [code](https://github.com/jgreener64/allopred)

# Software

Some software I have written or contributed to:
- [Molly.jl](https://github.com/JuliaMolSim/Molly.jl) - molecular simulation in Julia
- [Progres](https://github.com/jgreener64/progres) - fast protein structure searching
- [BioStructures.jl](https://github.com/BioJulia/BioStructures.jl) - a Julia package to read, write and manipulate macromolecular structures
- [mmterm](https://github.com/jgreener64/mmterm) - view proteins and trajectories in the terminal
- [Bio3DView.jl](https://github.com/jgreener64/Bio3DView.jl) - view macromolecular structures from Julia
- [pdb-benchmarks](https://github.com/jgreener64/pdb-benchmarks) - benchmarking common tasks on proteins in various languages and packages
- [cgdms](https://github.com/psipred/cgdms) - differentiable molecular simulation of proteins with a coarse-grained potential
- [DMPfold](https://github.com/psipred/DMPfold) and [DMPfold2](https://github.com/psipred/DMPfold2) - de novo protein structure prediction
- [AtomsBase.jl](https://github.com/JuliaMolSim/AtomsBase.jl) - a Julian abstract interface for atomic structures
- [Biopython](http://biopython.org) - biology tools in Python

There is more software associated with a few of the papers listed above.

# Talks

Talks and practical at SINEQ summer school, Ã‰cole des ponts ParisTech (Sep 2023) - [slides](https://docs.google.com/presentation/d/1SjzRi7jFbgFwP9kupwdtkxmr2x0OWjzzXwnoenCcCQg/edit?usp=sharing) - [notebook](https://github.com/noeblassel/SINEQSummerSchool2023/blob/main/notebooks/molly_average.ipynb)

Talk at Institute of Genetics and Cancer, University of Edinburgh (Mar 2023) - [slides](https://docs.google.com/presentation/d/11naV12vWIZ3TlmaX8-IB4nx6vUOwLXdcvZWABLPd09A/edit?usp=sharing)

Enzyme Conference 2023 (Feb 2023) - [video](https://www.youtube.com/watch?v=6G97jDVPlYc) - [slides](https://docs.google.com/presentation/d/1xsi2tbOJ3ceHu8-4izIMmo6M5EfnU1BrVwEtRBybZIM/edit?usp=sharing)

Crick Christmas Lecture 2022 (Dec 2022) - [slides](https://docs.google.com/presentation/d/1ilWqWmYNuuMeO-vhJqukBaGfW9FXa7WeFyykKuZmmzc/edit?usp=sharing)

JuliaMolSim minisymposium at JuliaCon 2022 (Jul 2022) - [video](https://youtu.be/trapn-yIv8g?t=1889) - [slides](https://docs.google.com/presentation/d/1Wkurom5G2N_qwqNueGwLCyYmpETT6dyeuW6UfUlrb5A/edit?usp=sharing)

8th Annual CCP-BioSim Conference invited talk (Jun 2022) - [slides](https://docs.google.com/presentation/d/1thJOrj8kAWKxLRWAEcDZ1tng0d2E4IWIyfJ4yot6wPs/edit?usp=sharing)

Lennard Jones Centre (May 2022) - [video](https://www.youtube.com/watch?v=0MLl6mQWtEQ) - [slides](https://docs.google.com/presentation/d/1jf9Zt58D1TWdc9p6YPxtGRNtlddvkSr8shWU5rTgIFM/edit?usp=sharing)

Scientific Machine Learning Webinar Series (Mar 2022) - [video](https://www.cmu.edu/aced/sciML.html) - [slides](https://docs.google.com/presentation/d/1Ukvmyt2tUPmGV6ryIgvuGMnJy5-b9awlmiPJfdMm4uQ/edit?usp=sharing)

Microsoft Research New England ML Ideas Lunch (Mar 2021) - [slides](https://docs.google.com/presentation/d/1FkSsUWMBQTbiGhc1u30qRcj25CslL5xnt9gAyqIcRjM/edit?usp=sharing)

Imperial College Computational Biology Society (Nov 2020) - [slides](https://docs.google.com/presentation/d/1zkJ8UwSwu2nsjIwHvIiTXL2z4fyxs_kJDjxjwoB8vBo/edit?usp=sharing)

KQ Codes Technical Social (Jan 2020) - [slides](https://docs.google.com/presentation/d/1wWdqTR1Ifls-psjcj-Euv6LY7Z5LWBei7VsiqPlB46E/edit?usp=sharing).

3DSIG meeting at ISMB (Jul 2019) - [video](https://www.youtube.com/watch?v=apv5YPM0HqY&list=PLmX8XnLr6zeHO2l7Gki56qjGxhhHYljin&index=17) - [slides](https://docs.google.com/presentation/d/1akuaq1QHbCVdxshmrtbJm6xssRZ3PwOYcR0PviU6YWg/edit?usp=sharing).

London Julia [Meetup group](https://www.meetup.com/London-Julia-User-Group/events/262441283) (Jun 2019) - [slides](https://docs.google.com/presentation/d/13KxvnwR-XFMFyUSth1hsXSoW0Zx5XLzds6fci7RUUDQ/edit?usp=sharing).

Friday Wrap talk at Institute of Structural and Molecular Biology (May 2019) - [slides](https://docs.google.com/presentation/d/1PAagp9hPXTT3ks1iOvFKOyw0TOD_ZyF9dSmvoVVudVg/edit?usp=sharing).

3DSIG meeting at ISMB (Jul 2016).

Bioinformatics London [Meetup group](https://www.meetup.com/Bioinformatics-London/events/230436340) (Apr 2016).

# Awards

Warren DeLano Structural Bioinformatics and Computational Biophysics Award for best presentation at 3DSIG 2016.

Awarded ISMB/ECCB Travel Fellowship to attend ISMB 2019 by 3DSIG and ISCB.

Prize for highest mark in MSc class (2014).

Runner-up prize for best presentation at Imperial College London Department of Life Sciences Research Day (2017).
